config/measureBTagEff.cut

```diff
@@ -1,7 +1,6 @@
 ##
 ## cuts used to measure b tag efficiencies on TT
 ## use analysis-like selections but no selections on BB mass
```

Comment:
@jonamotta since you're at it, actually you can also remove 'but no selection on BB mass'